### PR TITLE
routine

### PR DESCRIPTION
**Pull request by MaxMuecket**

### CODE REVIEW

**Review by tmh9x — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -20,12 +20,16 @@
...
     "stylelint": "^13.13.1",
     "stylelint-config-prettier": "^8.0.2",
     "stylelint-config-standard": "^22.0.0",
     "ts-node": "^9.1.1",
     "typescript": "^4.2.4",
     "vite": "^2.2.3"
   },
-  "dependencies": {}
+  "dependencies": {
+    "@types/inquirer": "^7.3.1",
```
Comment (christophersiem) [inline package.json]: please move to dev-dependency

File: src/server.ts [hunk 2]
```diff
@@ -1,3 +1,52 @@
...
+    const command = await askForCommand();
+
+    switch (command) {
+      case 'list':
+        {
+          const service = await selectService([
+            'Github',
+            'Codewars',
+            'Google',
+            '',
```
Comment (christophersiem) [inline src/server.ts]: empty string not needed here

**Review by christophersiem — APPROVED** [review]
lgtm 🤘🏽

File: src/server.ts (hunk 2 above)
Comment (MaxMuecket) [reply src/server.ts]: fixed

**Review by lmachens — APPROVED** [review]
gj

**Pull request merged**